dll/win32/browseui/CAutoComplete.h

```diff
@@ -153,6 +153,7 @@ class CAutoComplete
     static LPCWSTR GetWndClassName() { return WC_DROPDOWNW; }
     BOOL m_bInSetText; // this flag avoids subsequent action in WM_SETTEXT
     BOOL m_bInSelectItem; // this flag avoids subsequent action in LVN_ITEMCHANGED
+    BOOL m_bEditHasFocus; // This flag watchs the focus status
```

Comment:
either: This flag watch**e**s the focus status, or maybe even better just strip the commrnt, it doesn't transport any info imho beyond the well-chosen variables name:
```suggestion
    BOOL m_bEditHasFocus;
```